lib/rules/nested-interactive.json

```
@@ -0,0 +1,12 @@
+{
+  "id": "nested-interactive",
+  "matches": "nested-interactive-matches",
+  "tags": ["cat.keyboard", "best-practice"],
+  "metadata": {
+    "description": "Ensure interactive controls are not nested",
+    "help": "Nested interactive controls are not announced by screen readers"
```

Help text has to be the short one:

```suggestion
    "description": "Nested interactive controls are not announced by screen readers",
    "help": "Ensure interactive controls are not nested"
```